packages/docs-examples/components/select/select-icon/select-icon-example.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, Inject } from '@angular/core';
+import { KBQ_LOCALE_SERVICE, KbqLocaleService } from '@koobiq/components/core';
+import { KbqFormFieldModule } from '@koobiq/components/form-field';
+import { KbqIconModule } from '@koobiq/components/icon';
+import { KbqSelectModule } from '@koobiq/components/select';
+import { enUSLocaleDataSet } from '../en-US';
+import { esLALocaleDataSet } from '../es-LA';
+import { faIRLocaleDataSet } from '../fa-IR';
+import { ptBRLocaleDataSet } from '../pt-BR';
+import { ruRULocaleDataSet } from '../ru-RU';
+import { zhCNLocaleDataSet } from '../zh-CN';
+
+const localeDataSet = {
+    'en-US': enUSLocaleDataSet,
+    'zh-CN': zhCNLocaleDataSet,
+    'es-LA': esLALocaleDataSet,
+    'pt-BR': ptBRLocaleDataSet,
+    'ru-RU': ruRULocaleDataSet,
+    'fa-IR': faIRLocaleDataSet
+};
+
+/**
+ * @title Select icon
+ */
+@Component({
+    standalone: true,
+    selector: 'select-icon-example',
+    imports: [KbqFormFieldModule, KbqSelectModule, KbqIconModule],
+    template: `
+        <kbq-form-field style="width: 320px">
+            <kbq-select
+                [(value)]="selected"
+                [placeholder]="'Город'"
+            >
+                <kbq-select-matcher class="kbq-select__matcher">
+                    <i
+                        [color]="'contrast-fade'"
+                        style="margin-right: 8px"
+                        kbq-icon="kbq-globe_16"
+                    ></i>
+                    <span>
+                        <div class="kbq-select__match-container">
+                            <span class="kbq-select__matcher-text">
+                                {{ selected }}
+                            </span>
+                        </div>
+                    </span>
+                    <div class="kbq-select__arrow-wrapper">
+                        <i
+                            class="kbq-select__arrow"
+                            [color]="'contrast-fade'"
+                            kbq-icon="kbq-chevron-down-s_16"
+                        ></i>
+                    </div>
+                </kbq-select-matcher>
+                @for (option of options; track option) {
+                    <kbq-option [value]="option">
+                        <span [innerHTML]="option"></span>
+                    </kbq-option>
+                }
+            </kbq-select>
+        </kbq-form-field>
```

Comment:
а если иконку обозначить как kbqPrefix, сработает?
```suggestion
            <i         
                         kbqPrefix
                        [color]="'contrast-fade'"
                        style="margin-right: 8px"
                        kbq-icon="kbq-globe_16"
                    ></i>
            <kbq-select
                [(value)]="selected"
                [placeholder]="'Город'"
            >
                @for (option of options; track option) {
                    <kbq-option [value]="option">
                        <span [innerHTML]="option"></span>
                    </kbq-option>
                }
            </kbq-select>
        </kbq-form-field>
```

Reply:
это нужно при помощи префикса реализовывать

Reply:
<img width="539" alt="Снимок экрана 2024-11-23 в 16 46 31" src="https://github.com/user-attachments/assets/f4bb07f3-3a01-49b5-902d-37e54abb557b">
тут надо подумать

Reply:
на это нужно баг завести, это легко поправить